clusterversion: clean up unused 19.2/20.1 cluster versions

thoszhang opened the pull request:
clusterversion: remove commented-out old cluster versions

From now on, we're going to delete the entries for obsolete cluster
versions entirely instead of commenting them out.

Release note: None

----

clusterversion: clean up unused 19.2/20.1 cluster versions

Related to #47447.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/55453)
<!-- Reviewable:end -->

dt reviewed (approved):
would it make sense to just outright delete them instead of commenting them out?

Commit:
clusterversion: remove commented-out old cluster versions

From now on, we're going to delete the entries for obsolete cluster
versions entirely instead of commenting them out.

Release note: None

Commit:
clusterversion: clean up unused 19.2/20.1 cluster versions

Release note: None

lucy-zhang commented:
Agreed, I've deleted all our commented-out versions.

lucy-zhang commented:
TFTR!

bors r+
